.pylintrc

```diff
@@ -44,7 +44,10 @@ disable=I,
         duplicate-code,
         superfluous-parens,
         abstract-class-little-used,
-        too-few-public-methods
+        too-few-public-methods,
+        RP0401,
```

Comment:
What are these codes and can we swap them for the more descriptive keywords?

Reply:
I haven't found a way to replace them with keywoeds, but added comments describing what each code means.